third_party/coro/coro.c

```diff
@@ -103,13 +105,29 @@ coro_init (void)
 
   coro_transfer (new_coro, create_coro);
 
-#if __GCC_HAVE_DWARF2_CFI_ASM && __amd64
```

Comment:
Why do you change this ifdef? Why is it insufficient, and `ENABLE_BACKTRACE` should be used instead?

Reply:
Because after 7dc9fe4 this is feature-check is performed in CMake:
https://github.com/tarantool/tarantool/blob/63de77fd8119fd7f166ed2b70af083202db95263/cmake/compiler.cmake#L132-L149